src/platform/datapath_winkernel.c

```diff
@@ -962,6 +963,26 @@ CxPlatDataPathInitialize(
         goto Error;
     }
 
+    Status =
+        Datapath->WskProviderNpi.Dispatch->
+        WskControlClient(
+            Datapath->WskProviderNpi.Client,
+            WSK_TDI_BEHAVIOR,
+            sizeof(NoTdi),
+            &NoTdi,
+            0,
+            NULL,
+            NULL,
+            NULL);
+    if (QUIC_FAILED(Status)) {
+        QuicTraceEvent(
+            LibraryErrorStatus,
+            "[ lib] ERROR, %u, %s.",
+            Status,
+            "WskControlClient WSK_TDI_BEHAVIOR");
+        goto Error;
```

Comment:
Your wireguard PR ignores the error here. Should we just log and continue?

Reply:
Your call. On one hand, I kind of doubt that the MSDN mention of removing it will actually ever happen. On the other hand, if it does happen, I'd prefer the code to age well. But maybe you would be okay with just changing the code when the time comes. So that's up to you.

If you do think I should ignore the error, what's prefix would you want? `[ lib] WARN, `? Or keep it at `[ lib] ERROR`, but not `goto ERROR;` subsequently?

Reply:
You can just remove the `goto Error;` line. Keep the log as is.

Reply:
Works for me.